specifications/active/aRMT-1.15.0.yml

```diff
@@ -0,0 +1,263 @@
+  - type: AUDIO_VOCALISATION
+    doc: A speech questionnaire where users will record themselves speak the vowels displayed on the screen aloud.
+    topic: questionnaire_audio_vocalisation
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/audio_vocalisation/audio_vocalisation_armt.json
+  - type: EXACERBATION_DIARY
+    topic: questionnaire_exacerbation_diary
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/exacerbation_diary/exacerbation_diary_armt.json
+  - type: NUVOAIR_SPIROMETRY
+    topic: notification_nuvoair_spirometry
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/nuvoair_spirometry/nuvoair_spirometry_armt.json
+  - type: K_BUILD_PRIVATE
+    topic: questionnaire_k_build_private
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions-Private/master/questionnaires/k_build_private/k_build_private_armt.json
+  - type: COMPLETION_LOG
+    doc: Information about the completeness of each questionnaire.
+    topic: questionnaire_completion_log
+    value_schema: .monitor.questionnaire.QuestionnaireCompletionLog
+  - type: TIMEZONE
+    doc: Timezone information sent along with each questionnaire.
+    topic: questionnaire_timezone
+    value_schema: .monitor.application.ApplicationTimeZone
+  - type: APP_EVENT
+    doc: Questionnaire application interaction event.
+    topic: questionnaire_app_event
+    value_schema: .monitor.questionnaire.QuestionnaireApplicationInteractionEvent
+  - type: EPI_SEIZURE_DIARY
+    topic: questionnaire_epi_seizure_diary
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/epi_seizure_diary/epi_seizure_diary_armt.json
+  - type: EPI_EVENING_QUESTIONNAIRE
+    topic: questionnaire_epi_evening_questionnaire
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/epi_evening_questionnaire/epi_evening_questionnaire_armt.json
+  - type: EPI_WSAS
+    topic: questionnaire_epi_wsas
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/epi_wsas/epi_wsas_armt.json
+  - type: BP_QUESTIONNAIRE
```

Comment:
This is for another study but just realized yesterday from the protocol, the name is `bp`. So it might be better to change this to `type: BP`. What do you think @nivemaham?

Reply:
Is the type used anywhere? IIRC it is not, so we can change it as it will have no implications and it is nice to keep these consistent.

Reply:
Yes it's used by the `HLH-Pilot` study so I'm not sure if we can change.